### ISSUE TITLE
block-closing-brace-space-after False Positive

### ISSUE THREAD

**Issue by thegranddesign** [stylelint/stylelint#3700]
> Clearly describe the bug

I get `block-closing-brace-space-after: Unexpected whitespace after "}"` even though the config is set to `never`.  I'm assuming it's matching the newline which it should not since that is handled by `block-closing-brace-newline-after`.  There's literally no way to make this pass except to only have one block in the file.

> Which rule, if any, is the bug related to?

`block-closing-brace-space-after`

> What CSS is needed to reproduce the bug?

```scss
@font-face {
}

@font-face {
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "block-closing-brace-space-after": "never"
  }
}
```

> Which version of stylelint are you using?

`9.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint "**/*.css" --config myconfig.json`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's SCSS.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
2:2  ✖  Unexpected whitespace after "}"   block-closing-brace-space-after
```

**Comment by jeddy3** [comment]
@thegranddesign Thanks for the report and for using the template.

I believe the rule is behaving correctly. It's expecting no whitespace after all closing braces and there's a newline after the one on line number 2. 

Unless you wish to enforce the following code style:

```css
a { prop: value; } b { prop: value; }

c {
  prop: value;
  prop: value;
}

d {
  prop: value;
  prop: value;
}
```

With the `"always-single-line"` and `"always-multi-line"` options e.g.:

```
{
  "rules": {
    "block-closing-brace-newline-after": "always-multi-line"
    "block-closing-brace-space-after": "always-single-line"
  }
}
```

You'll probably want to turn off the `block-closing-brace-space-after` rule, either by not including it in your config or by setting its value to `null` e.g.

```json
{
  "rules": {
    "block-closing-brace-space-after": null,
  }
}
```

You can read more about how the stylistic rules work together [in this section of the docs](https://stylelint.io/user-guide/about-rules/#rules-work-together). 

Feel free to reopen this issue if the above doesn't resolve the issue for you.
